File: dto-mapping-benchmark/src/main/java/com/example/dtomappingbenchmark/v2/repository/UserRepository.java
package com.example.dtomappingbenchmark.v2.repository;

import com.example.dtomappingbenchmark.v2.entity.User;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.Optional;

@Repository("userRepositoryV2")
public interface UserRepository extends JpaRepository <User, Long>{
    Optional<User> findById(Long id);
}

File: object-mapping-benchmark-v5/src/main/kotlin/yu/selab/cj/objectmappingbenchmarkv5/user/repository/UserRepository.kt
package yu.selab.cj.objectmappingbenchmarkv5.user.repository

import org.springframework.data.jpa.repository.EntityGraph
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import yu.selab.cj.objectmappingbenchmarkv5.user.entity.User
import java.util.Optional

@Repository
interface UserRepository : JpaRepository<User, Long> {
}